### PR TITLE
[WIP] Ensure form validation

### PR DESCRIPTION
**Pull request by eatonphil**

### CODE REVIEW

File: src/components/Input.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes } from 'react';
+
+export default function Input(props) {
+  return (
+    <span className="Input">
```
Comment (mparke) [inline src/components/Input.js]: I'm all for having a default `Input`, but I'm in favor of keeping structure (in this case the wrapping span) separate . Thoughts on breaking out?

File: src/components/Input.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <input
+        {...props}
+        className={`form-control ${props.className}`}
+      />
+      {!props.label ? null : <label className="Input-label">{props.label}</label>}
```
Comment (mparke) [inline src/components/Input.js]: label should have `htmlFor=""` where possible, we should also keep this separate from input, as bootstrap grid classes can be used on labels within a form-group. Same idea as above comment, keeping structure separate.

File: src/components/Input.js (hunk 1 above)
Comment (eatonphil) [inline src/components/Input.js]: For React's sake, it needs to be wrapped in something. But it also needs to be for BEM's sake (so the whole thing gets the Input class).

**Comment by eatonphil** [comment]
Closing -- will break out into individual PRs

**Pull request closed without merging**